docs/source/quasi_dags.ipynb

```diff
@@ -0,0 +1,457 @@
+{
```

Comment:
<p>Good this is clear and I think useful. I think it's concise and we may want to avoid cluttering it with the discussion of ATE and LATE and ATC etc...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/321/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>👍🏻</p>